blocked-edges/4.18.0-rc.1-CMOSuddenStrictConfigValidation.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.18.0-rc.1
+from: ^4[.]18[.]0-ec[.][0-2][+].*$
+fixedIn: 4.18.0-rc.2
+url: https://issues.redhat.com/browse/OCPBUGS-42671
+name: CMOSuddenStrictConfigValidation
+message: |-
+    The Cluster Monitoring Operator (CMO) now validates the content of its ConfigMaps more strictly.
+
+    This may result in the operator being marked as `Degraded` and `Unavailable` while updating to a version with strict validation if any of the ConfigMaps contain invalid configurations.
+
+    The error messages should be clear enough to help identify and correct any eventual issues.
+matchingRules:
```

Comment:
From https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cincinnati-graph-data/6405/pull-ci-openshift-cincinnati-graph-data-master-validate-blocked-edges/1866071353514266624:

```
File "/cincinnati-graph-data/hack/validate-blocked-edges.py", line 84, in <module>
    validate_blocked_edges(directory='blocked-edges')
  File "/cincinnati-graph-data/hack/validate-blocked-edges.py", line 18, in validate_blocked_edges
    raise ValueError('invalid blocked edge {}: {}'.format(path, error)) from None
ValueError: invalid blocked edge blocked-edges/4.18.0-rc.1-CMOSuddenStrictConfigValidation.yaml: unrecognized keys in 'Always' rule: promql
```

Reply:
Dammit that's a bug in https://github.com/petr-muller/ota/tree/main/cmd/graph-extend-or-fix :monocle_face: